src/main/java/org/sagebionetworks/bridge/models/ParticipantData.java

```diff
@@ -0,0 +1,34 @@
+package org.sagebionetworks.bridge.models;
+// TODO: check if we need to refactor this into /models/reports/
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.sagebionetworks.bridge.dynamodb.DynamoParticipantData;
+import org.sagebionetworks.bridge.json.BridgeTypeName;
+
+@BridgeTypeName("ParticipantData")
+@JsonDeserialize(as= DynamoParticipantData.class)
+public interface ParticipantData extends BridgeEntity{
+
+    // TODO: I'm not sure what this is for but it's in ReportData
+    static TypeReference<ForwardCursorPagedResourceList<ParticipantData>> PAGED_REPORT_DATA = new TypeReference<ForwardCursorPagedResourceList<ParticipantData>>() {
```

Comment:
It's a type reference to convert JSON into the ForwardCursorPagedResourceList<ParticipantData> object. I think you'd only do this for tests, it's not something we receive through the API.